synth.py

```diff
@@ -32,24 +32,20 @@
    version,
    generator_args={
      "grpc-service-config": f"grafeas/{version}/grafeas_grpc_service_config.json",
-     "package-name":f"@google-cloud/grafeas"
+     "package-name":"@google-cloud/grafeas"
      },
-     proto_path=f'/grafeas/{version}',
-     extra_proto_files=['google/cloud/common_resources.proto'],
```

Comment:
We don't need common resources here?

Reply:
This is not intended, thank you for catching this. There is one resource of `project`. 
But I removed the common_resources.proto and run synthtool, it works fine somehow.